### PR TITLE
Solution of Game

### PR DESCRIPTION
**Pull request by svyshcho**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+            description=race_info.get("description")
+        )
+
+        skills_info = race_info.get("skills")
+        for skill_info in skills_info:
+            skill, _ = Skill.objects.get_or_create(
+                name=skill_info.get("name"),
+                bonus=skill_info.get("bonus"),
+                race=Race.objects.filter(name=race_info.get("name")).get()
+            )
```
Comment (EdAlekseiev) [inline main.py:25]: We must filter `Race` by name. Move `bonus` and `race ` to defaults.

File: main.py [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+            name=race_info.get("name"),
+            description=race_info.get("description")
+        )
+
+        skills_info = race_info.get("skills")
+        for skill_info in skills_info:
+            skill, _ = Skill.objects.get_or_create(
+                name=skill_info.get("name"),
+                bonus=skill_info.get("bonus"),
+                race=Race.objects.filter(name=race_info.get("name")).get()
```
Comment (EdAlekseiev) [inline main.py:24]: Don't search Race again, use `race` which was found above

File: main.py [hunk 3]
```diff
@@ -1,10 +1,44 @@
...
+            )
+        if info.get("guild"):
+            guild_info = info.get("guild")
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_info.get("name"),
+                description=guild_info.get("description")
```
Comment (EdAlekseiev) [inline main.py:30]: We must filter `Guild` by `name`. Move `description` to defaults.

File: main.py [hunk 4]
```diff
@@ -1,10 +1,44 @@
...
+            )
+        else:
+            guild = None
```
Comment (EdAlekseiev) [inline main.py:33]:
```suggestion
        guild = None
        if info.get("guild"):
            guild_info = info.get("guild")
            guild, _ = Guild.objects.get_or_create(
                name=guild_info.get("name"),
                description=guild_info.get("description")
            )
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (svyshcho) [reply main.py]: Done

File: main.py (hunk 2 above)
Comment (svyshcho) [reply main.py]: Done

File: main.py (hunk 4 above)
Comment (svyshcho) [reply main.py]: Done

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**